src/lib/routes/controller.ts

```diff
@@ -114,6 +125,8 @@ export default class Controller {
             this.useContentTypeMiddleware(options),
             this.useRouteErrorHandler(options.handler.bind(this)),
         );
+
+        this.app.use(options.path, checkOpenAPIValidationError());
```

Comment:
We can reuse this Open API validation handler for all services by using `options.path`.